stable/rocketchat/templates/deployment.yaml

```diff
@@ -0,0 +1,65 @@
+{{- if default "" .Values.host }}
```

Comment:
This is empty by default. When I install the chart with defaults only MongoDB comes up. It should be possible to install a chart with default values.